invenio_cli/cli/cli.py

```diff
@@ -88,8 +88,7 @@ def init(flavour, template, checkout):
 
         click.secho("Writing invenio-invenio_cli config file...", fg='green')
         saved_replay = cookiecutter_wrapper.get_replay()
-        instance_path = calculate_instance_path(project_dir)
```

Comment:
This was actually always returning an error that was hidden away by our `ProcessResponse`, so it doesn't even make sense to run it.